selenium/che-selenium-test/src/main/java/org/eclipse/che/selenium/pageobject/theia/TheiaIde.java

```diff
@@ -151,7 +153,11 @@ public void waitNotificationPanelClosed() {
   }
 
   public void waitTheiaIde() {
-    seleniumWebDriverHelper.waitVisibility(theiaIde, PREPARING_WS_TIMEOUT_SEC);
+    try {
+      seleniumWebDriverHelper.waitVisibility(theiaIde, PREPARING_WS_TIMEOUT_SEC);
+    } catch (WebDriverException ex) {
```

Comment:
Can we be more specific here, not to cover unrecoverable exceptions, but timeout exception?